ac1/_sources/projects/module_a.rst

```diff
@@ -14,60 +14,99 @@ In this project, you will design a survey to answer a research question of your
 own, and then share the results.
 
 Here are the steps you’ll need to complete.
+ 
+**Step 1: Choose a Research Question**
 
-1.  Form a group of no more than 4 people.
+In a group, decide on a research question. Your question should ask about the 
+relationship between two or more attributes. 
 
-    -   You may want to form a group with people who are in a similar major or
-        have similar interests as this may make question selection easier.
+Examples of good research questions include:
 
-2.  Decide on a research question with your group members. Your question should
-    ask about the relationship between **two or more** attributes.
+- Does handedness (left or right) have an impact on whether you played a sport
+  in high school? 
+- Do people who have dogs walk more than people who don’t?
 
-    -   Examples of a good research question: Does handedness (left or right)
-        have an impact on whether you played a sport in high school? Do people
-        who have dogs walk more than people who don’t?
-    -   Example of an inadequate question: What percent of people are
-        left-handed? (This question involves only one attribute.)
+An example of an inadequate research question would be: 
 
-3.  Design a survey with questions to help you answer your research question.
-    Remember to create clear questions that elicit specific answers.
+- What percent of people are left-handed? (This question only involves one 
+  attribute) 
 
-4.  Deploy your survey and collect **at least 30** responses.
+**To Do:**
 
-    -   Having trouble getting enough responses, or want more participants? Try
-        social media!
+- Form a group of no more than 4 people. (You may want to form a group with 
+  people who are in a similar major or have similar interests as this may make 
+  question selection easier.)
+- Choose a research question that asks about the relationship between two or 
+  more attributes 
```

Comment:
All bullet points should end with a period/question etc. A few are missing that, this one and one above. `- What percent of people are left-handed? (This question only involves one attribute)`